### DISCUSSION TITLE
Time to update supported browsers?

### DISCUSSION THREAD

**Discussion by gingerchew** [biati-digital/glightbox#432] [Ideas]
Since IE is no longer getting security updates and is excluded from MDN's compatibility table, I think it is safe to drop IE. It opens up a lot of options that can lead to a smaller package size (fewer polyfills, no more shimming for features that are now standard).

If we use the [defaults from browserslist](https://browsersl.ist/#q=defaults) that would give reasonable coverage across the board.

**Comment by biati-digital** [comment]
Older browser support will be removed in V4, it did help a lot to make the library a lot smaller.
